SCClassLibrary/Common/Quarks/Quarks.sc

```diff
@@ -161,6 +161,8 @@ Quarks {
 			},
 			prev = this.installed.detect({ |q| q.name == quark.name });
 
+		"Installing %".format(quark.name).inform;
```

Comment:
this should go after the next statement where it checks if its already installed.